### ISSUE TITLE
Chromecast support

### ISSUE THREAD

**Issue by edersong** [SamTV12345/PodFetch#217]
### Clear and concise description of the problem

I would suggest a Chromecast support where we be able to listen our podcasts in Nest devices

### Suggested solution

Chromecast compatibility

### Validations

- [X] Follow our [Code of Conduct](https://github.com/SamTV12345/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/SamTV12345/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by SamTV12345** [comment]
Good idea. I'll get back to you to test this feature. I don't have a Nest device unfortunately.

**Comment by edersong** [comment]
Count me in to test

**Comment by SamTV12345** [comment]
Are you still interested in this?

**Comment by edersong** [comment]
Yes, super interested!!

**Comment by bastgau** [comment]
Yes sure ! Very very interested !

**Comment by SamTV12345** [comment]
It's now implemented.
